modules/move-etcd-different-disk.adoc

```diff
@@ -10,15 +10,25 @@ You can move etcd from a shared disk to a separate disk to prevent or resolve pe
 
 The Machine Config Operator (MCO) is responsible for mounting a secondary disk for {product-title} {product-version} container storage.
 
+[NOTE]
+====
+This procedure only works with devices where the name starts with `sda` or `vda`.
```

Comment:
I don't see this limitation in the base64 code injected by the machine config.
The blkid runs against any sdX device on /dev/

Reply:
In the base64 code we have shown example of sda, but one can change that to /dev/vd*. We can write that as a note and users are free to change it. We can also show the base64 decoded part, if anyone wants to change according to their environment.